.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
.env.example file needs to be pushed so that any other dev knows what env variables are there to be set up